packages/datadog-plugin-grpc/src/util.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+const pick = require('lodash.pick')
+const log = require('../../dd-trace/src/log')
+
+module.exports = {
+  addMethodTags (span, path, kind) {
```

Comment:
It's already tested by the instrumentation for the client and server. Let's add tests for `getFilter` when we extract it since right now it's duplicated in a few places.

Reply:
I was thinking we could have a few test cases since right now we only test for when the path is `/test.TestService/<method>`, but I think it can get a bit more complicated than that, though.

We also definitely need tests for `getFilter` 👍